### ISSUE TITLE
Invalid Pypi package referenced in wiki

### ISSUE THREAD

**Issue by Zeerg** [dxa4481/truffleHog#183]
They package referenced here:

https://github.com/dxa4481/truffleHog/wiki/Install-instructions

Downloads a version of truffleHog that doesn't appear to be legit. Is this a troll or an old package? @sbehrens appears to be the owner of the pypi package. 

https://pypi.org/project/truffle-hog/
https://pypi.org/simple/truffle-hog/
https://files.pythonhosted.org/packages/60/e7/e72e873e38f6039ac461811868938c1a319e352e19b5090bbbc40d3d8d58/truffle-hog-0.1.tar.gz

```
class PostDevelopCommand(develop):
  """Post-installation for development mode."""

  def run(self):
    check_call("echo this host has been pwned", shell=True)
    develop.run(self)


class PostInstallCommand(install):
  """Post-installation for installation mode."""

  def run(self):
    check_call("echo this host has been pwned", shell=True)
    install.run(self)
```

**Comment by Zeerg** [comment]
@sbehrens are you real life or just fantasy?

**Comment by Zeerg** [comment]
Looks like 1.8k people have downloaded it https://pepy.tech/project/truffle-hog

**Comment by dxa4481** [comment]
This is known and authorized by me. We where doing research do see how easy it would be to backdoor packages given the fact many GitHub wiki's are world mutable.

TLDR it's easy. 

Thanks for spotting, no real backdoor Scott is a friend

**Comment by Zeerg** [comment]
Sounds fantastic!

**Comment by sbehrens** [comment]
Heh, I've been MIA for a quarter due to personal and professional things but i'm due to touch base with ya'll.  Hope things are going well.
